apps/src/StudioApp.js

```diff
@@ -1678,12 +1678,11 @@ StudioApp.prototype.displayFeedback = function(options) {
       project.getShareUrl();
   } catch (e) {}
 
-  if (
-    this.shouldDisplayFeedbackDialog_(
-      options.preventDialog,
-      options.feedbackType
-    )
-  ) {
+  options.useDialog = this.shouldDisplayFeedbackDialog_(
```

Comment:
I'm a bit nervous about the idea of treating the `options` object as mutable; it might be cleaner to do something like make the `shouldDisplayFeedbackDialog` method public

Reply:
Mind sharing a bit more detail about what's making you nervous? i.e. this function seems to be building the options object (see line 1670 for an example).

Reply:
oh, I didn't realize we were already mutating the object elsewhere. Nerves calmed, thanks!